python/pfnet/cpfnet.pyx

```diff
@@ -1258,16 +1596,16 @@ cdef class Branch:
 
     property sens_P_u_bound:
         """ Objective function sensitivity with respect to active power flow upper bound (float or array). """
-        def __get__(self): 
-            r = [cbranch.BRANCH_get_sens_P_u_bound(self._c_ptr,t) for t in range(self.num_periods)] 
+        def __get__(self):
+            r = [cbranch.BRANCH_get_sens_P_u_bound(self._c_ptr,t) for t in range(self.num_periods)]
             if self.num_periods == 1:
                 return AttributeFloat(r[0])
             else:
                 return np.array(r)
-                
+
```

Comment:
@ttinoco 

It is difficult to say for sure. I tried a couple of different things but what I've settled on for the past few months was to try and accept whatever format the original file uses (the editor tries to figure this out). One thing that always comes out odd is the mix of spaces and tabs. For instance, I will see that spaces have been used  for the first couple of indents and then tabs will be used.  I'm using Atom and it light weight editor that shouldn't change things automatically, but maybe when it automatically tries to figure out the tab vs. space usage something odd happens. I will try and look into it, or maybe disable the automatic setting and just try to be careful not delete the extra tabs/spaces I find as well.